### PR TITLE
👔 Clear secure storage after reinstalls

### LINKED ISSUE #42: Provide a way to reset the wrapper
The authentication data stored in secure storage is automatically synced by iOS into iCloud and is restored on new installs.
So once authenticated there seems to be now way to actually get rid of the authentication again.

Could we provide a way to log users out locally, without needing to end the session server side? So that I can clear out stale authentication data for new installs?

### PR DESCRIPTION
**Pull request by fa-fifi**
- Resolve #42 by clearing secure storage every time user reinstalls.

### CODE REVIEW

File: lib/src/wrapper.dart [hunk 1]
```diff
@@ -74,10 +74,18 @@ class KeycloakWrapper {
   }
 
   /// Initializes the user authentication state and refreshes the token.
-  void initialize() {
+  Future<void> initialize() async {
+    final prefs = SharedPreferencesAsync();
+    final hasRunBefore = await prefs.getBool('hasRunBefore') ?? false;
+
+    if (!hasRunBefore) {
+      _secureStorage.deleteAll();
```
Comment (lyio) [inline lib/src/wrapper.dart:82]: @fa-fifi this will logout everyone who is currently logged in, right?

Comment (fa-fifi) [reply lib/src/wrapper.dart:82]: What do u mean by everyone? Your keycloak allow multiple users login at the same time?

Comment (lyio) [reply lib/src/wrapper.dart:82]: I mean, everyone who is using the App and is updating.

Comment (fa-fifi) [reply lib/src/wrapper.dart:82]: Yes. Only on that particular device.

**Pull request merged**